lms/static/js/student_account/views/RegisterView.js

```diff
@@ -102,6 +154,94 @@
                     return this;
                 },
 
+                postRender: function() {
+                    var inputs = [
+                            $('#register-name'),
+                            $('#register-email'),
+                            $('#register-username'),
+                            $('#register-password'),
+                            $('#register-country')
+                        ],
+                        inputTipSelectors = ['tip error', 'tip tip-input'],
+                        focusinStyles = {
+                            position: 'relative',
+                            'padding-top': '0px',
+                            'padding-left': '0px',
+                            opacity: '1.0'
+                        },
+                        focusoutStyles = {
+                            position: 'absolute',
+                            'padding-top': '5px',
+                            'padding-left': '9px',
+                            opacity: '0.75',
+                            'z-index': '1'
+                        },
+                        onInputFocus = function() {
+                            // Apply on focus styles to input
+                            $(this).prev('label').css(focusinStyles);
+
+                            // Show each input tip
+                            $(this).siblings().each(function() {
+                                if (inputTipSelectors.includes($(this).attr('class'))) {
+                                    $(this).show();
```

Comment:
There are some performance issues with jQuery's ```show``` and ```hide``` methods so I would recommend updating to adding/removing CSS classes.